docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,11 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
-In order to use the {WebName} to analyze applications, you must create a project. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
+To use the {WebName} to analyze applications, you must create one or more projects. Each project is a way to group applications for analysis and specifies the settings to use during the analysis. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
 
```

Comment:
This paragraph can go into master.adoc under the heading.

Suggested rewrite:

You can create a project in the {WebName} to analyze your applications.

Each project groups the applications for a specific analysis, which you can configure with custom rules and labels.

The analysis process generates reports that describe the readiness of your applications for migration or modernization.